BDArmory/Damage/HitpointTracker.cs

```diff
@@ -1506,6 +1506,7 @@ void SetHullMass()
             if (isAI || ArmorPanel || ProjectileUtils.isMaterialBlackListpart(this.part))
             {
                 _hullConfigured = true;
+                part.gTolerance = !ArmorPanel ? 999 : 50; //50 for now, armor panels should probably either be determined by armor material, or arbitrary 'weld/mounting bracket' strength
```

Comment:
Wouldn't
```csharp
part.gTolerance = isAI ? 999 : ArmorPanel ? 50 : part.partInfo.partPrefab.gTolerance;
```
be better? Otherwise, this is setting things like the inflatable heatshield and radiator panels to have 999 g-tolerance.